### DISCUSSION TITLE
有必要迁移到小米官方 ha 插件吗

### DISCUSSION THREAD

**Discussion by yzlnew** [al-one/hass-xiaomi-miot#2089] [Q&A]
https://github.com/XiaoMi/ha_xiaomi_home

小米出了官方 ha 插件？看 README 支持中枢网关的本地控制，但是网关的固件还没有发布。

**Comment by charleypeng** [comment]
两个一起用

**Comment by VergilGao** [comment]
我在想，中枢网关内置了 mqtt broker 是不是可以不用官方集成登录，直接用 mqtt 集成自己写配置的方式来控制？这样就可以只有在配网时才用米家，日常使用时完全不需要公网连接了。
